docs/src/main/tut/kleisli.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: default
+title:  "Kleisli"
+section: "data"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/data/Kleisli.scala"
+scaladoc: "#cats.data.Kleisli"
+---
+# Kleisli
+Kleisli is a data type that will come in handy often, especially if you are working with monadic functions.
+
+## Functions
+One of the most useful properties of functions is that functions **compose**. That is, given a function
+`A => B` and a function `B => C`, we can compose them to create a new function `A => C`. It is through
```

Comment:
Maybe use another word instead of *compose* here, since you are defining it? Maybe *combine*, *chain together*, or something similar?